### PR TITLE
Add OpenStack Swift scaler e2e tests

### PR DESCRIPTION
**Pull request by pdominguite**
Signed-off-by: Pedro Felipe Dominguite <p.dominguite@sidi.org.br>

This PR adds end-to-end tests for the OpenStack Swift scaler.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] Changelog has been updated

Relates to #1462.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +41,7 @@
 - Check that metricNames are unique in ScaledObject ([#1390](https://github.com/kedacore/keda/pull/1390))
 - Serve OpenAPI spec from KEDA Metrics Apiserver ([#1512](https://github.com/kedacore/keda/pull/1512))
 - Support metrics with multiple dimensions and configurable metricValues on AWS Cloudwatch Scaler ([#1230](https://github.com/kedacore/keda/issues/1230))
+- Add end-to-end tests for OpenStack Swift scaler
```
Comment (zroubalik) [inline CHANGELOG.md:44]: Minor nit, no need to add a separate line, you can link this PR in the existing entry for OpenStack Swift scaler, ie update the line 24 of the Changelog to::
`- Introducing OpenStack Swift scaler ([#1342](https://github.com/kedacore/keda/issues/1342), [#1522](https://github.com/kedacore/keda/pull/1522))`

File: tests/.env [hunk 2]
```diff
@@ -5,3 +5,8 @@ AZURE_SUBSCRIPTION=
 AZURE_RESOURCE_GROUP=
 TEST_LOG_ANALYTICS_WORKSPACE_ID=
 TEST_STORAGE_CONNECTION_STRING=
+OS_SWIFT_URL=
+OS_TENANT_ID=
+OS_PASSWORD=
+OS_PROJECT_ID=
+OS_AUTH_URL=
```
Comment (zroubalik) [inline tests/.env:12]: We will need to fill these values right?

**Review by zroubalik — COMMENTED** [review]
Looking good, minor nit in Changelog

File: tests/.env (hunk 2 above)
Comment (pdominguite) [reply tests/.env:12]: The test will always look for these environment variables to run. I thought that every env variable used by tests should be declared there. Am I right or is it not necessary?

Comment (zroubalik) [reply tests/.env:12]: Yeah, the question was more about the OpenStack instance itself. Gonna ping you on Slack

**Review by zroubalik — APPROVED** [review]
LGTM, we need to figure out OpenStack Environment, that can be used to run the test against

**Comment by zroubalik** [comment]
Marking as draft, until we solve OpenStack Environment

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,7 +48,7 @@
 - Introducing InfluxDB scaler ([#1239](https://github.com/kedacore/keda/issues/1239))
 - Add Redis cluster support for Redis list and Redis streams scalers ([#1437](https://github.com/kedacore/keda/pull/1437))
 - Global authentication credentials can be managed using `ClusterTriggerAuthentication` objects ([#1452](https://github.com/kedacore/keda/pull/1452))
-- Introducing OpenStack Swift scaler ([#1342](https://github.com/kedacore/keda/issues/1342))
+- Introducing OpenStack Swift scaler ([#1342](https://github.com/kedacore/keda/issues/1342), [#1522](https://github.com/kedacore/keda/pull/1522))
```
Comment (zroubalik) [inline CHANGELOG.md:51]: Please update this entry, add it to the Unreleased to `### Other` section :)

**Review by zroubalik — CHANGES_REQUESTED** [review]

**Review by zroubalik — APPROVED** [review]

**Pull request merged**